detectors/opentelemetry_resource_detector_aws/src/otel_resource_aws_ecs.erl

```diff
@@ -0,0 +1,139 @@
+-module(otel_resource_aws_ecs).
```

Comment:
The naming convention here follows the naming convention in opentelemetry with [`otel_resource_app_env`](https://github.com/open-telemetry/opentelemetry-erlang/blob/main/apps/opentelemetry/src/otel_resource_env_var.erl) and [`otel_resource_env_var`](https://github.com/open-telemetry/opentelemetry-erlang/blob/main/apps/opentelemetry/src/otel_resource_env_var.erl)